apps/website/.vuepress/theme/components/Sidebar.vue

```diff
@@ -19,43 +19,51 @@
               <div
                 class="nav-group-children"
                 v-bind:class="{ 'is-expanded': states[index] || activePage.path.startsWith(item.path) }"
-                v-bind:style="{ height: states[index] ? `${item.children.length * 36}px` : '0' }"
+                v-bind:style="{
+                  /**
+                   * The `-1` is remove the additional whitespace added in `core-components` section
+                   */
+                  height: states[index] ? `${(item.children.length - betaComponents.length - 1) * 36}px` : '0',
+                }"
               >
-                <template v-for="childItem in item.children">
+                <template v-for="(childItem, index) in item.children">
                   <router-link
                     @focus.native="focusToggle(index)"
                     class="nav-link"
                     :to="childItem.path"
-                    v-if="childItem.type !== 'external'"
+                    v-if="childItem.type !== 'external' && isBeta(childItem) === false"
+                    :key="index"
                     v-bind:class="{
                       active: isItemActive(childItem),
                     }"
                   >
                     <span class="nav-text">
                       {{ childItem.title }}
-                      <cds-icon
+                      <!-- <cds-icon
```

Comment:
If we don't need this could it be removed?

Reply:
I am predicting this would be needed as some point in future. I would say - lets keep it so we don't rewrite it from scratch if we need it back